calour/filtering.py

```diff
@@ -1,6 +1,8 @@
 # calour functions for filtering samples/observations
 # functions should call reorder_samples() / reorder_obs()
 
+import calour as ca
+
 
```

Comment:
i want to have all the calour functions under one place (like importing numpy...) why not do it?